first-party/jib-layer-filter-extension-maven/src/test/java/com/google/cloud/tools/jib/maven/extension/layerfilter/JibLayerFilterExtensionTest.java

```diff
@@ -71,6 +111,15 @@ private static FileEntriesLayer buildLayer(String layerName, List<String> filePa
     return filter;
   }
 
+  private static Dependency mockDependency(String sourcePath, String artifactId) {
+    File file = mock(File.class);
+    when(file.toPath()).thenReturn(Paths.get(sourcePath));
+    Artifact artifact = mock(Artifact.class);
+    when(artifact.getArtifactId()).thenReturn(artifactId);
+    when(artifact.getFile()).thenReturn(file);
```

Comment:
Probably this can just be
```suggestion
    when(artifact.getFile()).thenReturn(Paths.get(sourcePath).toFile());
```
or
```suggestion
    when(artifact.getFile()).thenReturn(new File(sourcePath));
```

Reply:
Ok.